src/super_gradients/examples/regseg_transfer_learning_example/regseg_transfer_learning_example.py

```diff
@@ -52,4 +51,7 @@
                                                                             last_img_idx_in_batch=4)],
                 }
 
-trainer.train(train_params)
+trainer.train(training_params=train_params,
```

Comment:
please put the model first in the params (just to keep it clean)

Reply:
fixed in https://github.com/Deci-AI/super-gradients/pull/364/commits/b1b57b1e73601102855d9e3b9c31691489cb98b2